cobigen/cobigen-tsplugin/src/main/java/com/devonfw/cobigen/tsplugin/merger/constants/Constants.java

```diff
@@ -21,6 +21,11 @@
     /** Export statement regex */
     public static final String EXPORT_REGEX = "export\\s+([^\\s]+)";
 
+    /**
+     * Path of the executable file where the server of the external process is stored
+     */
+    public static final String EXE_PATH = "src/test/resources/testdata/externalserver/nestserver.exe";
+
     /** We want to check whether it is a real export statement, not like the type "export class a" */
     public static final Map<String, Boolean> NOT_EXPORT_TYPES;
```

Comment:
btw. is this needed anymore? The whole merging is done in ts or is there any logic also in java?

Reply:
but wouldn't this be solved if you make the beautify also on the language server?

Reply:
The beautify process is already done on the server side, however it still messes up the imports. For instance, from this:

`import {bla, ble} from bla/path`

It converts it to:

```TypeScript
import {
bla,
ble
}
from
bla/path
```

Splitting the imports from the body could be done on the server side. However, I don't see the point of doing it like that, as we have it already implemented on the Java side + we reduce the data to send over IPC..